code/modules/mob/living/carbon/human/species/yautja/_species.dm

```diff
@@ -53,8 +53,8 @@
 		/mob/living/carbon/human/proc/mark_panel,
 	)
 
-	knock_down_reduction = 4
-	stun_reduction = 4
+	knock_down_reduction = 1.3
+	stun_reduction = 1.3
```

Comment:
```
/mob/living/carbon/human/GetKnockDownDuration(amount)
	. = ..()
	var/skill_resistance = skills ? (skills.get_skill_level(SKILL_ENDURANCE)-1)*0.08 : 0
	var/final_reduction = (1 - skill_resistance) / species.knock_down_reduction
	return . * final_reduction
```
1.3 is a weird choice why not 1 or 2

Reply:
It's easier to achieve proper balance when you use decimals

Reply:
because 1 was too little and 2 was too fast

Reply:
1.3 works perfectly to how it was before thats why i settled on it, 1.5 is also fine